{"0e513447efa92164a2c9240852530ae6":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":3},"items":[{"uid":"e2d8a671aeb2b8d1","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/e2d8a671aeb2b8d1","status":"passed","time":{"start":1658061697022,"stop":1658061697023,"duration":1}},{"uid":"170fe6a69eeff9aa","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/170fe6a69eeff9aa","status":"passed","time":{"start":1658061223093,"stop":1658061223094,"duration":1}},{"uid":"ed3b00a5578dd2af","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/ed3b00a5578dd2af","status":"passed","time":{"start":1658060689958,"stop":1658060689958,"duration":0}}]},"ee7adf2efbaaa135b16aeb889b9d3efd":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":3},"items":[{"uid":"5d5897e4ce66938e","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/5d5897e4ce66938e","status":"passed","time":{"start":1658061697028,"stop":1658061697028,"duration":0}},{"uid":"2073bcd1aac8fe2a","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/2073bcd1aac8fe2a","status":"passed","time":{"start":1658061223098,"stop":1658061223098,"duration":0}},{"uid":"e3ef5505379a0126","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/e3ef5505379a0126","status":"passed","time":{"start":1658060689963,"stop":1658060689963,"duration":0}}]},"0ec84876de384187e37f98293337dc17":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"8f4dc3cf571bb49d","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/8f4dc3cf571bb49d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x562bad366b13 <unknown>\n#1 0x562bad16d688 <unknown>\n#2 0x562bad191f1f <unknown>\n#3 0x562bad18d5aa <unknown>\n#4 0x562bad1c864a <unknown>\n#5 0x562bad1c27a3 <unknown>\n#6 0x562bad1980ea <unknown>\n#7 0x562bad199225 <unknown>\n#8 0x562bad3ae2dd <unknown>\n#9 0x562bad3b22c7 <unknown>\n#10 0x562bad39822e <unknown>\n#11 0x562bad3b30a8 <unknown>\n#12 0x562bad38cbc0 <unknown>\n#13 0x562bad3cf6c8 <unknown>\n#14 0x562bad3cf848 <unknown>\n#15 0x562bad3e9c0d <unknown>\n#16 0x7f73eb816609 <unknown>","time":{"start":1658061702936,"stop":1658061703602,"duration":666}},{"uid":"cadb09faf920f83","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/cadb09faf920f83","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55b6215d6b13 <unknown>\n#1 0x55b6213dd688 <unknown>\n#2 0x55b621401f1f <unknown>\n#3 0x55b6213fd5aa <unknown>\n#4 0x55b62143864a <unknown>\n#5 0x55b6214327a3 <unknown>\n#6 0x55b6214080ea <unknown>\n#7 0x55b621409225 <unknown>\n#8 0x55b62161e2dd <unknown>\n#9 0x55b6216222c7 <unknown>\n#10 0x55b62160822e <unknown>\n#11 0x55b6216230a8 <unknown>\n#12 0x55b6215fcbc0 <unknown>\n#13 0x55b62163f6c8 <unknown>\n#14 0x55b62163f848 <unknown>\n#15 0x55b621659c0d <unknown>\n#16 0x7fe790083609 <unknown>","time":{"start":1658061225356,"stop":1658061226040,"duration":684}},{"uid":"55e6ee7b42c1cfcf","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/55e6ee7b42c1cfcf","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e753132b13 <unknown>\n#1 0x55e752f39688 <unknown>\n#2 0x55e752f5df1f <unknown>\n#3 0x55e752f595aa <unknown>\n#4 0x55e752f9464a <unknown>\n#5 0x55e752f8e7a3 <unknown>\n#6 0x55e752f640ea <unknown>\n#7 0x55e752f65225 <unknown>\n#8 0x55e75317a2dd <unknown>\n#9 0x55e75317e2c7 <unknown>\n#10 0x55e75316422e <unknown>\n#11 0x55e75317f0a8 <unknown>\n#12 0x55e753158bc0 <unknown>\n#13 0x55e75319b6c8 <unknown>\n#14 0x55e75319b848 <unknown>\n#15 0x55e7531b5c0d <unknown>\n#16 0x7f9b3670a609 <unknown>","time":{"start":1658060692220,"stop":1658060692898,"duration":678}}]},"030d0f8c101a152ea8a3293a775caf95":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"c59a25f08a765021","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/c59a25f08a765021","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x560282276b13 <unknown>\n#1 0x56028207d688 <unknown>\n#2 0x5602820a1f1f <unknown>\n#3 0x56028209d5aa <unknown>\n#4 0x5602820d864a <unknown>\n#5 0x5602820d27a3 <unknown>\n#6 0x5602820a80ea <unknown>\n#7 0x5602820a9225 <unknown>\n#8 0x5602822be2dd <unknown>\n#9 0x5602822c22c7 <unknown>\n#10 0x5602822a822e <unknown>\n#11 0x5602822c30a8 <unknown>\n#12 0x56028229cbc0 <unknown>\n#13 0x5602822df6c8 <unknown>\n#14 0x5602822df848 <unknown>\n#15 0x5602822f9c0d <unknown>\n#16 0x7f17dee4c609 <unknown>","time":{"start":1658061702142,"stop":1658061702830,"duration":688}},{"uid":"53eac364b58623ab","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/53eac364b58623ab","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x5632be4c8b13 <unknown>\n#1 0x5632be2cf688 <unknown>\n#2 0x5632be2f3f1f <unknown>\n#3 0x5632be2ef5aa <unknown>\n#4 0x5632be32a64a <unknown>\n#5 0x5632be3247a3 <unknown>\n#6 0x5632be2fa0ea <unknown>\n#7 0x5632be2fb225 <unknown>\n#8 0x5632be5102dd <unknown>\n#9 0x5632be5142c7 <unknown>\n#10 0x5632be4fa22e <unknown>\n#11 0x5632be5150a8 <unknown>\n#12 0x5632be4eebc0 <unknown>\n#13 0x5632be5316c8 <unknown>\n#14 0x5632be531848 <unknown>\n#15 0x5632be54bc0d <unknown>\n#16 0x7f0f4a7b9609 <unknown>","time":{"start":1658061224562,"stop":1658061225253,"duration":691}},{"uid":"7ae995ff43da0129","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/7ae995ff43da0129","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c6f652bb13 <unknown>\n#1 0x55c6f6332688 <unknown>\n#2 0x55c6f6356f1f <unknown>\n#3 0x55c6f63525aa <unknown>\n#4 0x55c6f638d64a <unknown>\n#5 0x55c6f63877a3 <unknown>\n#6 0x55c6f635d0ea <unknown>\n#7 0x55c6f635e225 <unknown>\n#8 0x55c6f65732dd <unknown>\n#9 0x55c6f65772c7 <unknown>\n#10 0x55c6f655d22e <unknown>\n#11 0x55c6f65780a8 <unknown>\n#12 0x55c6f6551bc0 <unknown>\n#13 0x55c6f65946c8 <unknown>\n#14 0x55c6f6594848 <unknown>\n#15 0x55c6f65aec0d <unknown>\n#16 0x7f4035002609 <unknown>","time":{"start":1658060691422,"stop":1658060692106,"duration":684}}]},"91eb24abe5818f2b6bed1123d9018b41":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":3},"items":[{"uid":"48173940b3604aad","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/48173940b3604aad","status":"passed","time":{"start":1658061697026,"stop":1658061697026,"duration":0}},{"uid":"7b433eb6f0efa238","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/7b433eb6f0efa238","status":"passed","time":{"start":1658061223096,"stop":1658061223097,"duration":1}},{"uid":"7e92d8e9579b5ca1","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/7e92d8e9579b5ca1","status":"passed","time":{"start":1658060689961,"stop":1658060689962,"duration":1}}]},"0139c44e2d86c93c634c3d6d6688f462":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"d5b2bf48493e3359","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/d5b2bf48493e3359","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e902437b13 <unknown>\n#1 0x55e90223e688 <unknown>\n#2 0x55e902262f1f <unknown>\n#3 0x55e90225e5aa <unknown>\n#4 0x55e90229964a <unknown>\n#5 0x55e9022937a3 <unknown>\n#6 0x55e9022690ea <unknown>\n#7 0x55e90226a225 <unknown>\n#8 0x55e90247f2dd <unknown>\n#9 0x55e9024832c7 <unknown>\n#10 0x55e90246922e <unknown>\n#11 0x55e9024840a8 <unknown>\n#12 0x55e90245dbc0 <unknown>\n#13 0x55e9024a06c8 <unknown>\n#14 0x55e9024a0848 <unknown>\n#15 0x55e9024bac0d <unknown>\n#16 0x7fe6cac3d609 <unknown>","time":{"start":1658061697030,"stop":1658061701984,"duration":4954}},{"uid":"4a040da2fab8b2ec","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/4a040da2fab8b2ec","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x564cf2afbb13 <unknown>\n#1 0x564cf2902688 <unknown>\n#2 0x564cf2926f1f <unknown>\n#3 0x564cf29225aa <unknown>\n#4 0x564cf295d64a <unknown>\n#5 0x564cf29577a3 <unknown>\n#6 0x564cf292d0ea <unknown>\n#7 0x564cf292e225 <unknown>\n#8 0x564cf2b432dd <unknown>\n#9 0x564cf2b472c7 <unknown>\n#10 0x564cf2b2d22e <unknown>\n#11 0x564cf2b480a8 <unknown>\n#12 0x564cf2b21bc0 <unknown>\n#13 0x564cf2b646c8 <unknown>\n#14 0x564cf2b64848 <unknown>\n#15 0x564cf2b7ec0d <unknown>\n#16 0x7fd2576d4609 <unknown>","time":{"start":1658061223099,"stop":1658061224411,"duration":1312}},{"uid":"bb88a16afe2b4d17","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/bb88a16afe2b4d17","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c33feb1b13 <unknown>\n#1 0x55c33fcb8688 <unknown>\n#2 0x55c33fcdcf1f <unknown>\n#3 0x55c33fcd85aa <unknown>\n#4 0x55c33fd1364a <unknown>\n#5 0x55c33fd0d7a3 <unknown>\n#6 0x55c33fce30ea <unknown>\n#7 0x55c33fce4225 <unknown>\n#8 0x55c33fef92dd <unknown>\n#9 0x55c33fefd2c7 <unknown>\n#10 0x55c33fee322e <unknown>\n#11 0x55c33fefe0a8 <unknown>\n#12 0x55c33fed7bc0 <unknown>\n#13 0x55c33ff1a6c8 <unknown>\n#14 0x55c33ff1a848 <unknown>\n#15 0x55c33ff34c0d <unknown>\n#16 0x7fa9e5d24609 <unknown>","time":{"start":1658060689965,"stop":1658060691260,"duration":1295}}]},"6022a14518909447816a44551c9542d0":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":3},"items":[{"uid":"25ed80bbe24ed545","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/4//#testresult/25ed80bbe24ed545","status":"passed","time":{"start":1658061697025,"stop":1658061697025,"duration":0}},{"uid":"af76b4c04a97196d","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/3//#testresult/af76b4c04a97196d","status":"passed","time":{"start":1658061223095,"stop":1658061223095,"duration":0}},{"uid":"4cc8fde1c19698ff","reportUrl":"https://vapolonov.github.io/qa_guru_python_7/2//#testresult/4cc8fde1c19698ff","status":"passed","time":{"start":1658060689960,"stop":1658060689960,"duration":0}}]}}